home-data/src/main/java/com/softserveinc/ita/homeproject/homedata/repository/InvitationRepository.java

```diff
@@ -0,0 +1,32 @@
+package com.softserveinc.ita.homeproject.homedata.repository;
+
+import java.time.LocalDateTime;
+import java.util.List;
+import java.util.Set;
+
+import org.springframework.data.jpa.repository.Modifying;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.CrudRepository;
+import org.springframework.data.repository.query.Param;
+import org.springframework.stereotype.Repository;
+import org.springframework.transaction.annotation.Transactional;
+
+import com.softserveinc.ita.homeproject.homedata.entity.Invitation;
+
+@Repository
+public interface InvitationRepository extends CrudRepository<Invitation, Long> {
+
+    @Transactional
```

Comment:
Why did you wrap it in a transaction? What operations does it include?

Reply:
While using annotation Query for updating or deleting it was required. But I've changed it. Thanks